fuzz/src/chanmon_consistency.rs

```diff
@@ -157,7 +157,7 @@ impl TestChainMonitor {
 			logger,
 			keys,
 			persister,
-			latest_monitors: Mutex::new(HashMap::new()),
+			latest_monitors: Mutex::new(new_hash_map()),
```

Comment:
When should we use functions like `new_hash_map`? I see it's also used in some non-fuzzing code currently.